src/cpuid.cpp

```diff
@@ -30,9 +30,6 @@ POSSIBILITY OF SUCH DAMAGE.
 
 */
 
-#include "libtorrent/config.hpp"
-#include <cstring>
-
 #include "libtorrent/aux_/cpuid.hpp"
 
 #if defined _MSC_VER && TORRENT_HAS_SSE
```

Comment:
isn't TORRENT_HAS_SSE defined by config.hpp? i.e. it will always be false with this patch